workers/unity/Packages/io.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -30,15 +30,17 @@ public static void Build()
                 var args = CommandLineArgs.FromCommandLine();
 
                 // Parse command line arguments
-                var buildTargetFilter = CommandlineParser.GetBuildTargetFilter(args);
-                var wantedWorkerTypes = CommandlineParser.GetWorkerTypesToBuild(args);
-                var scriptImplementation = CommandlineParser.GetScriptingImplementation(args);
-                var buildEnvironment = CommandlineParser.GetBuildEnvironment(args);
-                var targetIOSSdkVersion = CommandlineParser.GetTargetIOSSdk(args);
+                var buildContextFilter = new BuildContextFilter
+                {
+                    WantedWorkerTypes = CommandlineParser.GetWorkerTypesToBuild(args),
+                    BuildEnvironment = CommandlineParser.GetBuildEnvironment(args),
+                    ScriptImplementation = CommandlineParser.GetScriptingImplementation(args),
+                    BuildTargetFilter = CommandlineParser.GetBuildTargetFilter(args),
+                    IOSSdkVersion = CommandlineParser.GetTargetIOSSdk(args)
+                };
```

Comment:
Move this into a `BuildContextFilter.FromCommandLine(args);`? 

Also `CommandlineParser.GetWorkerTypesToBuild` and others can be moved in there too if they aren't used elsewhere (I don't believe they are).